.github/workflows/integration.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
 
       - name: Test
         run: |
-          RUST_LOG=info cargo test -p integration-tests -- --nocapture
+          RUST_LOG=tantivy=off,info cargo test -p integration-tests -- --nocapture --test-threads=3
```

Comment:
Isnt this (optimal) number derived from #cores on the CPU anyway?

Reply:
This was a desperate hack to get CI to pass the `bombastic_search` test. I couldn't reproduce locally.